src/main/java/com/controller/MoviesController.java

```diff
@@ -0,0 +1,27 @@
+package com.controller;
+
+import com.model.movies.Movies;
+import com.service.MoviesService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "/simple-movies-db", produces = MediaType.APPLICATION_JSON_VALUE)
+public class MoviesController {
+
+    @Autowired
+    private MoviesService moviesService;
+
+    private static final String PATH = "/Users/famin/MoviesDB/simple-movies-database/src/main/resources/static/movies.json";
```

Comment:
Can you find a way to perhaps use a relative path? This application wouldn't currently run on my machine

Reply:
Resolved this by using relative paths